cartographer_ros/cartographer_ros/tf_bridge.cc

```diff
@@ -30,23 +30,30 @@ TfBridge::TfBridge(const string& tracking_frame,
 
 std::unique_ptr<::cartographer::transform::Rigid3d> TfBridge::LookupToTracking(
     const ::cartographer::common::Time time, const string& frame_id) const {
-  ::ros::Duration timeout(lookup_transform_timeout_sec_);
+  tf2::Duration timeout(lookup_transform_timeout_sec_ * 1000000000.0);
```

Comment:
If you're going to use a float anyways, maybe the `1e9` syntax would be easier to read.